src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -278,14 +278,29 @@ fn runtime_dll_dist(rust_root: &Path, target: TargetSelection, builder: &Builder
         return;
     }
 
-    let (bin_path, _) = get_cc_search_dirs(target, builder);
-
-    let mut rustc_dlls = vec!["libwinpthread-1.dll"];
-    if target.starts_with("i686-") {
-        rustc_dlls.push("libgcc_s_dw2-1.dll");
+    let (bin_path, libs_path) = get_cc_search_dirs(target, builder);
+
+    let mut rustc_dlls = vec![];
+    // windows-gnu and windows-gnullvm require different runtime libs
+    if target.ends_with("windows-gnu") {
+        rustc_dlls.push("libwinpthread-1.dll");
+        if target.starts_with("i686-") {
+            rustc_dlls.push("libgcc_s_dw2-1.dll");
+        } else {
+            rustc_dlls.push("libgcc_s_seh-1.dll");
+        }
     } else {
-        rustc_dlls.push("libgcc_s_seh-1.dll");
+        rustc_dlls.push("libunwind.dll");
     }
+    // FIXME(#144565): Remove this whole `let ...`
```

Comment:
I don't understand the FIXME, it also refers to some other PR?

Reply:
Good catch, I meant #144656